### DISCUSSION TITLE
Not sending notification after failed backup v6.8

### DISCUSSION THREAD

**Discussion by aanyszek** [spatie/laravel-backup#1045] [Q&A]
Hello, 
i have issue: when remote backup fail with connection problem notification is not sending. 

> $ php artisan  backup:run  --only-db
> Starting backup...
> Dumping database topq...
> Determining files to backup...
> Zipping 1 files and directories...
> Created zip containing 1 files and directories. Size is 408.88 KB
> Copying zip to disk named remote-backups...
> Copying zip failed because: Could not login with connection: xxx:21, username: xxxx.
> Sending notification failed
> Backup completed!

I expect to receive notification message with connection error to email or slack.

**Comment by Cannonb4ll** [comment]
Had the same issue, Laravel version 7 with this as an exception:

```
Target [Illuminate\Contracts\Mail\Factory] is not instantiable while building [Illuminate\Notifications\Channels\MailChannel].
```

**Comment by pms07** [comment]
Same issue occurred on production system Ubuntu 18.04 with PHP 7.2 but perfectly working fine on local system while testing (local system: windows 10 with PHP - 7.2
